lib/arca/report.rb

```diff
@@ -51,8 +54,8 @@ def conditionals_count
       number_of_unique_conditionals(model.analyzed_callbacks_array)
     end
 
-    delegate :lines_between_count, :external_callbacks_count,
-      :external_targets_count, :external_conditionals_count, :to => :model
+    def_delegators :model, :lines_between_count, :external_callbacks_count,
```

Comment:
I usually do :@model if the ivar is always set. Faster as ivar lookup is faster than method cal + ivar lookup. Minor, but probably a good habit long term.